packages/storybook/src/nimble/button/button.mdx

```diff
@@ -1,45 +1,269 @@
 import { Canvas, Meta, Controls, Title } from '@storybook/addon-docs/blocks';
+import ButtonAnatomy from './images/Button.png';
 import ContentHiddenDocs from '../patterns/button/content-hidden-docs.mdx';
 import StylingDocs from '../patterns/button/styling-docs.mdx';
 import ComponentApisLink from '../../docs/component-apis-link.mdx';
 import { buttonTag } from '@ni/nimble-components/dist/esm/button';
 import { anchorButtonTag } from '@ni/nimble-components/dist/esm/anchor-button';
 import * as buttonStories from './button.stories';
+import { NimbleIconKey } from '@ni/nimble-react/dist/esm/icons/key';
+import { NimbleButton } from '@ni/nimble-react/dist/esm/button';
+import { NimbleSelect } from '@ni/nimble-react/dist/esm/select';
+import { NimbleListOption } from '@ni/nimble-react/dist/esm/list-option';
 
 <Meta of={buttonStories} />
 <Title of={buttonStories} />
 
-Per [W3C](https://www.w3.org/WAI/ARIA/apg/patterns/button/) - A button is a
-widget that enables users to trigger an action or event, such as submitting a
-form, opening a dialog, canceling an action, or performing a delete operation.
+## Overview
 
-Consider <Tag name={anchorButtonTag} /> if you want a button that triggers
-navigation to a URL.
+A button is a widget that enables users to trigger an action or event, such as
+submitting a form, opening a dialog, canceling an action, or performing a delete
+operation. _Source: [W3C](https://www.w3.org/WAI/ARIA/apg/patterns/button/)_
 
-<Canvas of={buttonStories.outlineButton} />
+<TableOfContents
```

Comment:
I was a bit confused by the visual design of the ToC initially. Is it a breadcrumb with arrows instead of slashes between the items? Is it links with arrows next to them? What does the arrow signify again? (My first guess was external link but I guess not). Why isn't there an arrow next to the last link?

<img width="799" height="54" alt="image" src="https://github.com/user-attachments/assets/c040c8f9-675f-4ab0-9751-509ee3dc4d51" />

I sort of figured out answers to those by playing with it but I'm not sure I see how this is better than the ToC we had before.